tests/sieve.c

```diff
@@ -0,0 +1,84 @@
+/* create a baseline for optimization comparison */
```

Comment:
Run `cd tests && clang-format -i sieve.c` to indent.

Reply:
I don't have clang-format

Reply:
No problem. You can add `(cd tests && clang-format sieve.c)` and copy the results from [GitHub Actions](https://github.com/jserv/amacc/actions).

Reply:
Not sure how this would work, since I am unfamiliar with all tools involved.